### DISCUSSION TITLE
VS Code extension?

### DISCUSSION THREAD

**Discussion by shaddyhm** [tinymanorg/tealish#67] [General]
Is it possible to have this extension in the VS market place? 
open source it?
Update readme to include the link instead of the dropbox link?

I just learned about tealish. This is very exciting. Nice work! Thank you.

**Comment by fergalwalsh** [comment]
Hi @shaddyhm, there is a more full featured extension in the works that incorporates the syntax highlighter + completions and jump-to-definition etc: https://github.com/thencc/TealishVSCLangServer. I'd highly recommend this. (you can install from the package in that repo)
I was waiting for it to be added to the VS code market place so I could add a link to in the docs.

@youraerials any update on your plans for releasing this?
